index.js

```diff
@@ -87,7 +88,7 @@ You can check by going to https://${servicenowHost}/nav_to.do?uri=%2Fu_standard_
 
     // Set outputs for GitHub Actions
     setOutput('change-sys-id', result.change_sys_id)
-    setOutput('work-start', result.workStart)
+    setOutput('work-start', convertServicenowTimestampFromMountainToUtc(result.workStart))
```

Comment:
does service now understand UTC time then? Or would the standard RFC's interpret the start time as 6 hours earlier than they actually happened?

Reply:
As part of this API call, ServiceNow gives us a `workStart` timestamp that's more-or-less the current time, in Utah.

On the other end, when updating the standard RFC to close it, the endpoint [accepts a `work_start` date](https://github.com/byu-oit/github-action-end-standard-change/blob/31b7f4293fcc5020c50ed8134cbc046f792c971f/index.js#L31) (that it interprets as the "actual" work start date, since that might've been slightly later than when you started the RFC if you were a human), only it now assumes that timestamp is in UTC. So, the output here has to be shifted 6/7 hours to match the input on the other end.

Currently, as in before we make the change from this PR, it's interpreting the start time as being 6 hours earlier than it actually happened. We're trying to fix that.

Reply:
That seems really odd to me that SN is so inconsistent in this way. But se la vie 🤷‍♂️